lapack/gonum/dgesvd.go

```diff
@@ -26,7 +26,7 @@ const noSVDO = "dgesvd: not coded for overwrite"
 // jobU and jobVT are options for computing the singular vectors. The behavior
 // is as follows
 //  jobU == lapack.SVDAll       All m columns of U are returned in u
-//  jobU == lapack.SVDInPlace   The first min(m,n) columns are returned in u
+//  jobU == lapack.SVDStore     The first min(m,n) columns are returned in u
```

Comment:
For me "in-place" and "overwrite" are synonyms, "store" indicates that the result is, well, stored in a designated destination.

Reply:
This seems reasonable.